app/views/payments/_form.html.erb

```diff
@@ -0,0 +1,33 @@
+<%= form_with(model: payment) do |form| %>
+  <% if payment.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(payment.errors.count, "error") %> prohibited this post from being saved:</h2>
+
+      <ul>
+        <% payment.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div>
+    <%= form.text_field :name, placeholder: "Name" %>
+  </div>
+
+  <div>
+    <%= form.number_field :amount, placeholder: "Amount" %>
+  </div>
+
+  <div class="checkbox-wrap">
+    <%= form.label :categories, style: "display: block" %>
+
+    <%= form.collection_check_boxes :category_ids, @categories, :id, :name %>
+    <br>
+     <%# <%= form.select :category_id, @categories.collect { |c| [c.name, c.id] } %>
+  </div>
+
+  <div>
+    <%= form.submit %>
+  </div>
```

Comment:
- [ ] When I click on **Create Payment** button without filling out the required fields, I got the error on the screenshot below instead of validation messages. As requested above :arrow_up: let's kindly resolve the issue.

![Screenshot from 2023-04-29 15-26-19](https://user-images.githubusercontent.com/63087491/235308339-af7d5b0c-f940-47b5-bc1d-1f1ef9ff40f8.png)

Reply:
- [ ] [OPTIONAL] I noticed that when I add a payment name that is below 3 characters I get an error flash method, you can make the error flash method to be more detailed like `name too short` instead of `something went wrong` :+1: 
![Screenshot from 2023-04-29 19-29-37](https://user-images.githubusercontent.com/66097371/235318985-2df784c6-838c-4934-bc95-f6fe93334e13.png)